OpenAI-DotNet/OpenAIClientSettings.cs

```diff
@@ -23,6 +23,7 @@ public OpenAIClientSettings()
             BaseRequest = $"/{ApiVersion}/";
             BaseRequestUrlFormat = $"https://{ResourceName}{BaseRequest}{{0}}";
             UseOAuthAuthentication = true;
+            PooledConnectionLifetime = TimeSpan.FromMinutes(15);
```

Comment:
Why 15 min?

Reply:
I couldn't find any real information on what a sensible default would be. Microsoft's examples used 15 minutes.